### PR TITLE
Docker: implement whanos images

### PR DESCRIPTION
**Pull request by majent4**

### CODE REVIEW

File: jenkins/images/python/Dockerfile.standalone [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM whanos-python
+
+SHELL [ "/bin/sh", "-c" ]
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip3 install --no-cache-dir -r requirements.txt
```
Comment (Cavonstavant) [inline jenkins/images/python/Dockerfile.standalone:9]:
```suggestion
ONBUILD pip3 install --no-cache-dir -r requirements.txt
```

File: jenkins/images/python/Dockerfile.base [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.10-alpine3.17@sha256:c69d5dfc1909d980947efa9d8f646bce3a31cf67109d3b9952d166d5d1ce2f2d
+
+WORKDIR /app
+
+SHELL [ "/bin/sh", "-c" ]
```
Comment (Cavonstavant) [inline jenkins/images/python/Dockerfile.base:5]: Useless as it's the default behavior of a Dockerfile

File: jenkins/images/python/Dockerfile.base [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.10-alpine3.17@sha256:c69d5dfc1909d980947efa9d8f646bce3a31cf67109d3b9952d166d5d1ce2f2d
+
+WORKDIR /app
+
```
Comment (Cavonstavant) [inline jenkins/images/python/Dockerfile.base:4]:
```suggestion

RUN pip3 install --no-cache-dir -r requirements.txt
```

File: jenkins/images/javascript/Dockerfile.standalone [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM whanos-javascript
+
+SHELL [ "/bin/sh", "-c" ]
+
+COPY . /app
+
+WORKDIR /app
+
+RUN npm i
```
Comment (Cavonstavant) [inline jenkins/images/javascript/Dockerfile.standalone:9]:
```suggestion
ONBUILD npm i
```

File: jenkins/images/python/Dockerfile.base (hunk 2 above)
Comment (majent4) [reply jenkins/images/python/Dockerfile.base:6]: Ok, resolved

**Review by Cavonstavant — APPROVED** [review]

**Review by osvegn — APPROVED** [review]

**Pull request merged**